### ISSUE TITLE
Can't login after upgrading

### ISSUE THREAD

**Issue by Johandrex** [plan-player-analytics/Plan#2605]
### Describe the issue
After upgrading from 5.4 build 1722 to 5.5 build 1951 I could no longer login. on the webserver.

Steps:
1. Go to the site "https://analytics.powercraft.network/"
2. Attempt to sign in with correct login information
3. You're not getting access, no errors found.

### Plugin versions
5.5 build 1951 on both the Proxy and Spigot servers.

### Additional information
Configuration file: https://hastebin.com/qagicezoji.yaml
Site configuration from apache: https://hastebin.com/esehapisen.apache

**Comment by AuroraLS3** [comment]
Do you have cloudflare in front of Plan?

**Comment by Johandrex** [comment]
Nope, hosting it myself on Linux, Ubuntu with Apache

**Comment by AuroraLS3** [comment]
Okay I'll have to look into it - could you post your reverse proxy configuration so I can figure things out

**Comment by Johandrex** [comment]
Thanks :), this one? https://hastebin.com/esehapisen.apache

**Comment by AuroraLS3** [comment]
Yep that's it. I'll try write some tests for reverse proxy setup logins to reproduce the problem

**Comment by AuroraLS3** [comment]
Could you try if clearing browser cache fixes the issue?

**Comment by AuroraLS3** [comment]
No worries, I was really stumped until I got my hands on a bugged system and when it didn't happen on my login I suspected cache issues.

There may be something that has changed with the cookie handling so it doesn't read cookies from the old version properly, so not your fault at all :)
